diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 0ee7c90..8398bcb 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -5,6 +5,7 @@ namespace AppBundle\Entity;
 
 use AppBundle\Repository\GenusRepository;
 use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
 use Doctrine\Common\Collections\Criteria;
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Component\Validator\Constraints as Assert;
@@ -90,40 +91,37 @@ class Genus
         $this->genusScientists = new ArrayCollection();
     }
 
-    public function getId()
+    public function getId(): ?int
     {
         return $this->id;
     }
 
-    public function getName(): string
+    public function getName(): ?string
     {
         return $this->name;
     }
 
-    public function setName(string $name)
+    public function setName(?string $name): void
     {
         $this->name = $name;
     }
 
-    /**
-     * @return SubFamily
-     */
-    public function getSubFamily()
+    public function getSubFamily(): ?SubFamily
     {
         return $this->subFamily;
     }
 
-    public function setSubFamily(SubFamily $subFamily = null)
+    public function setSubFamily(SubFamily $subFamily = null): void
     {
         $this->subFamily = $subFamily;
     }
 
-    public function getSpeciesCount()
+    public function getSpeciesCount(): ?int
     {
         return $this->speciesCount;
     }
 
-    public function setSpeciesCount($speciesCount)
+    public function setSpeciesCount(?int $speciesCount): void
     {
         $this->speciesCount = $speciesCount;
     }
@@ -140,50 +138,50 @@ class Genus
         return;
     }
 
-    public function getUpdatedAt()
+    public function getUpdatedAt(): \DateTimeInterface
     {
         return new \DateTime('-'.rand(0, 100).' days');
     }
 
-    public function setIsPublished($isPublished)
+    public function setIsPublished(bool $isPublished): void
     {
         $this->isPublished = $isPublished;
     }
 
-    public function getIsPublished()
+    public function getIsPublished(): bool
     {
         return $this->isPublished;
     }
 
     /**
-     * @return ArrayCollection|GenusNote[]
+     * @return Collection|GenusNote[]
      */
-    public function getNotes()
+    public function getNotes(): Collection
     {
         return $this->notes;
     }
 
-    public function getFirstDiscoveredAt()
+    public function getFirstDiscoveredAt(): ?\DateTimeInterface
     {
         return $this->firstDiscoveredAt;
     }
 
-    public function setFirstDiscoveredAt(\DateTime $firstDiscoveredAt = null)
+    public function setFirstDiscoveredAt(\DateTime $firstDiscoveredAt = null): void
     {
         $this->firstDiscoveredAt = $firstDiscoveredAt;
     }
 
-    public function getSlug()
+    public function getSlug(): ?string
     {
         return $this->slug;
     }
 
-    public function setSlug($slug)
+    public function setSlug(?string $slug): void
     {
         $this->slug = $slug;
     }
 
-    public function addGenusScientist(GenusScientist $genusScientist)
+    public function addGenusScientist(GenusScientist $genusScientist): void
     {
         if ($this->genusScientists->contains($genusScientist)) {
             return;
@@ -194,7 +192,7 @@ class Genus
         $genusScientist->setGenus($this);
     }
 
-    public function removeGenusScientist(GenusScientist $genusScientist)
+    public function removeGenusScientist(GenusScientist $genusScientist): void
     {
         if (!$this->genusScientists->contains($genusScientist)) {
             return;
@@ -206,9 +204,9 @@ class Genus
     }
 
     /**
-     * @return ArrayCollection|GenusScientist[]
+     * @return Collection|GenusScientist[]
      */
-    public function getGenusScientists()
+    public function getGenusScientists(): Collection
     {
         return $this->genusScientists;
     }
@@ -216,7 +214,7 @@ class Genus
     /**
      * @return \Doctrine\Common\Collections\Collection|GenusScientist[]
      */
-    public function getExpertScientists()
+    public function getExpertScientists(): Collection
     {
         return $this->getGenusScientists()->matching(
             GenusRepository::createExpertCriteria()
