diff --git a/app/Resources/views/genus/show.html.twig b/app/Resources/views/genus/show.html.twig
index a476631..ad0d706 100644
--- a/app/Resources/views/genus/show.html.twig
+++ b/app/Resources/views/genus/show.html.twig
@@ -17,6 +17,8 @@
                 <dd>{{ genus.funFact|markdownify }}</dd>
                 <dt>Recent Notes</dt>
                 <dd>{{ recentNoteCount }}</dd>
+                <dt>Diet</dt>
+                <dd>{{ recentlyAte }}</dd>
 
                 <dt>Lead Scientists</dt>
                 <dd>
diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index 7725aed..9215f9c 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -105,10 +105,13 @@ class GenusController extends Controller
         $recentNotes = $em->getRepository('AppBundle:GenusNote')
             ->findAllRecentNotesForGenus($genus);
 
+        $food = ['shrimp', 'clams', 'lobsters', 'shark'];
+
         return $this->render('genus/show.html.twig', array(
             'genus' => $genus,
             'funFact' => $funFact,
-            'recentNoteCount' => count($recentNotes)
+            'recentNoteCount' => count($recentNotes),
+            'recentlyAte' => $genus->feed($food),
         ));
     }
 
diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 8398bcb..292418e 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -220,4 +220,13 @@ class Genus
             GenusRepository::createExpertCriteria()
         );
     }
+
+    public function feed(array $food): string
+    {
+        if (count($food) === 0) {
+            return sprintf('%s is looking at you in a funny way', $this->getName());
+        }
+
+        return sprintf('%s recently ate: %s', $this->getName(), implode(', ', $food));
+    }
 }
