diff --git a/src/AppBundle/Controller/MainController.php b/src/AppBundle/Controller/MainController.php
index 8510c5f..a63f5d3 100644
--- a/src/AppBundle/Controller/MainController.php
+++ b/src/AppBundle/Controller/MainController.php
@@ -26,12 +26,10 @@ class MainController extends Controller
             }
 
             throw new NoCookiesLeft();
-        } catch (NoCookieForYou $e) {
-            $whisper = sprintf('Crazy Dave whispered "%s"', $e->getMessage());
-        } catch (NoCookiesLeft $e) {
+        } catch (NoCookieForYou | NoCookiesLeft $e) {
             $whisper = sprintf('Crazy Dave whispered "%s"', $e->getMessage());
         }
 
-        return new Response('<html><body>'.$whisper.'</body></html>');
+        return new Response(sprintf('<html><body>%s</body></html>', $whisper));
     }
 }
