diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index 92f96e1..e5ff8bb 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -21,10 +21,11 @@ class GenusController extends Controller
     public function typesExampleAction()
     {
         $genus = new Genus();
-        $genus->setName(4);
-
-        dump($genus);die;
+        $genus->setName('Octopus');
 
+        dump($genus);
+        dump($genus->getName());
+        die;
     }
 
     /**
diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 45e2f77..77c6b25 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -97,6 +97,7 @@ class Genus
 
     public function getName()
     {
+        return 5;
         return $this->name;
     }
 
