diff --git a/src/AppBundle/Controller/GenusController.php b/src/AppBundle/Controller/GenusController.php
index 302a262..575ec81 100644
--- a/src/AppBundle/Controller/GenusController.php
+++ b/src/AppBundle/Controller/GenusController.php
@@ -15,6 +15,17 @@ use Symfony\Component\HttpFoundation\Response;
 class GenusController extends Controller
 {
     /**
+     * @Route("/types")
+     */
+    public function typesExampleAction()
+    {
+        $genus = new Genus();
+        $genus->setName(4);
+
+        var_dump($genus);die;
+    }
+
+    /**
      * @Route("/genus/new")
      */
     public function newAction()
diff --git a/src/AppBundle/Entity/Genus.php b/src/AppBundle/Entity/Genus.php
index 9d95676..81583a9 100644
--- a/src/AppBundle/Entity/Genus.php
+++ b/src/AppBundle/Entity/Genus.php
@@ -99,7 +99,7 @@ class Genus
         return $this->name;
     }
 
-    public function setName($name)
+    public function setName(string $name)
     {
         $this->name = $name;
     }
